mw-webapp/src/logic/wayPage/jobTags/jobTag/JobTag.module.scss

```diff
@@ -1,20 +1,26 @@
 @import "src/styles/variables";
 
-$JobTagMaxWidth: 120px;
+$JobTagMaxWidth: 100px;
+$JobTagSmallMaxWidth: 80px;
+$JobTagSmallFontSize: 0.8rem;
```

Comment:
shoudl we really use specific loca font size in rem? 
100px is not enough for many long words, lets keep it 120px

Reply:
why we should not use rem? PLease explain

Reply:
80px is not enough for any 2 words, could we use 120px? Please play with numbers

Reply:
80px is not enough for any 2 words, could we use 120px?